sound/soc/intel/boards/sof_rt5682.c

```diff
@@ -295,11 +293,16 @@ static int sof_rt5682_codec_init(struct snd_soc_pcm_runtime *rtd)
 	struct snd_soc_component *component = asoc_rtd_to_codec(rtd, 0)->component;
 	struct snd_soc_jack *jack;
 	int extra_jack_data;
-	int ret;
+	int ret, mclk_freq;
+
+	mclk_freq = sof_dai_get_mclk(rtd);
 
 	/* need to enable ASRC function for 24MHz mclk rate */
 	if ((sof_rt5682_quirk & SOF_RT5682_MCLK_EN) &&
-	    (sof_rt5682_quirk & SOF_RT5682_MCLK_24MHZ)) {
+	    ((sof_rt5682_quirk & SOF_RT5682_MCLK_24MHZ) ||
```

Comment:
If you managed to remove this quirk from CML, then it can also be removed from  WhiskeyLake which is the same hardware. And that would mean there are zero users of that quirk...

Reply:
Not familiar with WHL so kept it untouched. Will remove the quirk. Thanks.